src/domain/valueObject/secureAccessPublicKeyName.go

```diff
@@ -0,0 +1,32 @@
+package valueObject
+
+import (
+	"errors"
+	"regexp"
+
+	voHelper "github.com/goinfinite/os/src/domain/valueObject/helper"
+)
+
+const SecureAccessPublicKeyNameRegex string = `^[\w@\-_]{6,32}$`
```

Comment:
The key name may start with a - _ or @? That's weird. I think you should go with `^[A-Za-z0-9][\w@\-_]{5,32}$` instead.